RxCocoa/Common/Observable+Bind.swift

```diff
@@ -84,4 +83,30 @@ extension ObservableType {
             rxFatalErrorInDebug("Binding error: \(error)")
         })
     }
+
+    /**
+    Bind emitted elements to the referenced key path of a specific `Root` object.
```

Comment:
Since the object is passed as a parameter, maybe 
```suggestion
    Bind emitted elements to the referenced key path of the given `Root` object.
```
What do you think?

Reply:
Don’t think it changes the meaning that much but sure, thanks